src/Components/Web/src/PublicAPI.Shipped.txt

```diff
@@ -334,7 +334,6 @@ Microsoft.AspNetCore.Components.Web.Virtualization.Virtualize<TItem>.Placeholder
 Microsoft.AspNetCore.Components.Web.Virtualization.Virtualize<TItem>.Placeholder.set -> void
 Microsoft.AspNetCore.Components.Web.Virtualization.Virtualize<TItem>.RefreshDataAsync() -> System.Threading.Tasks.Task!
 Microsoft.AspNetCore.Components.Web.Virtualization.Virtualize<TItem>.Virtualize() -> void
-Microsoft.AspNetCore.Components.Web.WebEventCallbackFactoryEventArgsExtensions
```

Comment:
The plan is to mark this type as Obsolete and remove it in a future release of ASP.NET Core.